app/src/main/res/values/strings.xml

```diff
@@ -8,6 +8,13 @@
     <string name="no_classes">Free :)</string>
     <string name="calendar">Calendar</string>
     <string name="week_schedule">Week schedule</string>
+    <string name="password">Password</string>
+    <string name="identifier">ID</string>
+    <string name="login">Login</string>
+    <string name="edziekanat">e-Dziekanat</string>
+    <string name="field_cannot_be_empty">Field cannot be empty</string>
+    <string name="incorrect_error">Upłynał czas bezczynności serwisu lub wystąpił brak dostępu do zasobu, proszę zalogować się ponownie</string>
```

Comment:
This string should be [marked as non-translatable](https://sites.google.com/a/android.com/tools/recent/non-translatablestrings) (to exclude it from translation warnings and [pseudolocalization](http://blog.danlew.net/2015/04/06/pseudolocalization-visiting-androids-bizarro-world/)) and then removed from `strings-pl.xml` (where it's exact copy lives).